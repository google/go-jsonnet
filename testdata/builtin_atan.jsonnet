local tolerance = (import 'tolerance.libsonnet').tolerance;
tolerance(std.atan(1))
