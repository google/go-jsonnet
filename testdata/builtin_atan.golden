0.78539816339744828
